backend/FwLite/FwLiteProjectSync.Tests/SyncTests.cs

```diff
@@ -84,6 +84,26 @@ internal static EquivalencyOptions<Entry> SyncExclusions(EquivalencyOptions<Entr
         return options;
     }
 
+    internal static void AssertSnapshotsAreEquivalent(ProjectSnapshot expected, ProjectSnapshot actual)
+    {
+        var excludeOrderTypes = new[] { typeof(Sense), typeof(ExampleSentence), typeof(ComplexFormComponent), };
+        var excludeIds = new[] { typeof(ComplexFormComponent), typeof(WritingSystem) };
+        actual.Should().BeEquivalentTo(expected,
+            options =>
+                options
+                    //DO NOT exclude things which are just not yet supported, otherwise you will run into a similar issue as seen in https://github.com/sillsdev/languageforge-lexbox/issues/1912
```

Comment:
You've taken a very robust approach.
It forces us to only merge/release fields that are fully supported.
Which generally sounds attractive, so I'm in favor of merging this.

I just think it's worth noting, that at the time we appreciated being able to ship partial PublishIn support.

If we wanted to do that again we'd have to make a bunch of exceptions and only test the project-snapshot that we actually write to disk and massage it into matching the CRDT data. But...that would be decently messy, so maybe we never want to do that 😆.

Reply:
I think we decided that this would be fine in our team meeting, so I'll take out this aggressive comment. I do think I'll add a comment which points to the issue where we're now using the crdt api